trio/_core/_local.py

```diff
@@ -1,24 +1,25 @@
 # Runvar implementations
+import attr
+
 from . import _run
 
 from .._util import Final
 
 
+@attr.s(eq=False, hash=False, slots=True)
 class _RunVarToken:
     _no_value = object()
 
-    __slots__ = ("_var", "previous_value", "redeemed")
+    _var = attr.ib()
+    previous_value = attr.ib(default=_no_value)
```

Comment:
We are ok with the `.__init__()` parameter name changing from `value` to `previous_value`?  This might be private enough that passing tests suffice.  I thought [`attr.ib()`](https://www.attrs.org/en/20.3.0/api.html#attr.ib) had something for this but I don't see it now.

This isn't a requested change, just making sure it's considered.

Reply:
I considered it and came to the same conclusions you did: it's private API and the tests pass so it's okay.

Reply:
Yeah I also considered it a better internal API:

`_RunVarToken(var=value, value=old_value)`